content-repo/gendocs.py

```diff
@@ -88,6 +90,29 @@ def timestamped_print(*args, **kwargs):
 PACKS_SCRIPTS_PREFIX = 'Scripts'
 PACKS_PLAYBOOKS_PREFIX = 'Playbooks'
 
+SERVICE_ACCOUNT = os.getenv('GCP_SERVICE_ACCOUNT')
+
+
+def create_service_account_file():
+    """
+    Create a service account json file from the circle variable.
+    """
+    service_account_file = tempfile.NamedTemporaryFile(delete=False, mode='w', suffix='.json')
+    json.dump(json.loads(SERVICE_ACCOUNT), service_account_file)
```

Comment:
why do you need to load and then dump? why not just write `SERVICE_ACCOUNT` directly to `service_account_file`?